README.adoc

```diff
@@ -418,14 +431,21 @@ bootstrap address by running the following command:
 oc get kafka kafka-cluster -o=jsonpath='{.status.listeners[?(@.type=="plain")].bootstrapServers}{"\n"}'
 ```
 
-The [hotspot=inventory file=1]`inventory` application is configured similarly to the [hotspot=system file=1]`system`
-application. Additionally, the `inventory` application includes the [hotspot=service file=1]`service` and
+You will see output similar to the following example:
+
+[role='no_copy']
+```
+kafka-cluster-kafka-bootstrap.guide.svc:9092
+```
+
+The [hotspot=inventory file=1]`inventory` microservice is configured similarly to the [hotspot=system file=1]`system`
+microservice. Additionally, the `inventory` microservice includes the [hotspot=service file=1]`service` and
 [hotspot=expose file=1]`expose` parameters. The [hotspot=servicePort file=1]`service.port` parameter specifies which port
-is exposed by the container, allowing the application to be accessed from outside the container. To access the application
+is exposed by the container, allowing the microservice to be accessed from outside the container. To access the microservice
 from outside of the cluster, it must be exposed by setting the [hotspot=expose file=1]`expose` parameter to `true`.
-Exposing the application lets the Operator automatically create and configure routes for external access to your application.
+Exposing the microservice lets the Operator automatically create and configure routes for external access to your microservice.
```

Comment:
suggested edit:

After you expose the microservice, the Operator automatically creates and configures routes for external access to your microservice.